projects/Canary/ClientApp/src/components/HomeScreen.js

```diff
@@ -31,7 +31,7 @@ export class HomeScreen extends Component {
           <CanaryCard
             title='Vital Records Death Reporting'
             subtitle='VRDR STU2'
-            link='https://canary-v4.fhir.nvss.cdc.gov'
```

Comment:
this should still link to https://canary-v4.fhir.nvss.cdc.gov 

Le wants to have one version of Canary get sent out to jurisdictions (the one in this repo), but have the VRDR card link to the version of Canary using vrdr-dotnet 4.x until we have full support for vrdr-dotnet 5.x